tests/setup_test.go

```diff
@@ -1928,3 +1929,96 @@ func TestUserOrgs(t *testing.T) {
 		t.Fatalf("user orgs mismatch (-want +got):\n%s", diff)
 	}
 }
+
+func TestTaskTimeout(t *testing.T) {
+	config := `
+      {
+        runs: [
+          {
+            name: 'run01',
+            tasks: [
+              {
+                name: 'task01',
+                runtime: {
+                  containers: [
+                    {
+                      image: 'alpine/git',
+                    },
+                  ],
+                },
+				task_timeout: 15000000000,
```

Comment:
Use a simple time string: `15s`

Reply:
@sgotti ok but I should to implement a custom Duration because there is error when I unmarshal jsonnet into time.Duration; and jsonnet don't support Duration type. Do you like better I implement a custom Duration or I leave task_timeout in number type?

Reply:
@sgotti another way is tu use a "%d" into config variable and next `config = fmt.Sprintf(config, 15*time.Second)` it works

Reply:
See https://github.com/agola-io/agola/pull/331/files/13048c190a594797dcec7ae9dff218627360c551#diff-54c7c1af5fa8d5db4dc49f0e8e80e93ba2b1183ba4d5c9e2e5729e6deae6a3cd

You should implement a custom duration (see https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/duration.go) 

Please do this in another PR. Also update the services config fields and for backward compatibility we should also handle times without any unit and cosider them nanoseconds.

Reply:
I clicked to the first link I see the message "We went looking everywhere, but couldn’t find those commits.".
If I leave task_timeout_interval in the config as string, and I do time.ParseDuration next is a good idea? (I just pushed so you can see)

Reply:
I'll use a dedicated Duration type.